tests/data_helpers_test.py

```diff
@@ -0,0 +1,24 @@
+# pylint: disable=not-callable, no-member, invalid-name, line-too-long, wildcard-import, unused-wildcard-import, missing-docstring
+import unittest
+
+import torch
+import e3nn.data_helpers as dh
+from e3nn import rs
+
+
+class Tests(unittest.TestCase):
```

Comment:
I think we should not use this `unittest` anymore and do like https://github.com/rusty1s/pytorch_sparse/blob/master/test/test_transpose.py

Reply:
Do you want this addressed here or do you want us to rewrite all tests in this style in another PR?

Reply:
Just change it here and one day I will change them all in a separate PR

Reply:
Does that require changing something about how CI is run? I just use python3 to run tests with unittest but for pytest you have to run pytest...